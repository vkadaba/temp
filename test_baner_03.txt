a : bool;
{
    b = + - * e b c d;
    c = / b a;
}
