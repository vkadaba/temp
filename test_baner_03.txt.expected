a: bool #
b: bool #
e: bool #
c: bool #
d: bool #
